### PR TITLE
Update Coronavirus Status to Step 5

### PR DESCRIPTION
**Pull request by awflwafl**
## Description

It's time to go to _Step 5_.

[Latest restrictions for Northern Ireland](https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you) allow indoor gatherings in non-domestic settings (events), so let's update our guidance for members and visitors to reflect it.

> Indoor gatherings (excluding private homes)
> To determine the maximum number of people permitted to attend an indoor gathering, in a non-domestic setting, the organiser or operator must carry out a risk assessment.
> 
> Indoor gatherings of 15 people or fewer do not need a risk assessment.
> 
> The organiser must also take all reasonable measures to limit the risk of transmission of the coronavirus.
> 
> A person responsible for organising a gathering must, if requested to do so by a relevant person, provide:
> 
> *  a copy of the risk assessment
> and
> * an account of the reasonable measures taken.
> It is important that indoor spaces are well ventilated at all times, by leaving doors and windows open. Frequently touched surfaces, such as door handles, should be cleaned regularly.
> 
> Good hand hygiene should be encouraged and social distancing between households maintained.

### CODE REVIEW

**Comment by awflwafl** [comment]
Have to wait on 👍 from @FarsetLabs/directors before we can apply, in case there is a reason to hold.

File: coronavirus.md [hunk 1]
```diff
@@ -2,18 +2,20 @@
 
 Information for Farset Labs members and visitors during the Coronavirus pandemic, in line with [government guidelines](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/our-approach-to-decision-making-summary-120520.pdf).
 
-#### **Status: Step 3**
+#### **Status: Step 5**
 
-**The space is open to members that went through COVID-19 onboarding and guests under their supervision, but not public events, and all hackerspace meetings are remote.**
+**The space is open to members and small public events with organisers that went through COVID-19 onboarding who can bring in guests under their supervision, and all hackerspace meetings are back in the space with remote participation supported.**
```
Comment (andrewbolster) [inline coronavirus.md:7]: I think we should define what 'small' means, possibly even being specific to just the event space, for instance 

```suggestion
**The space is open to members and small public events (<20 participants in the Event Space) with organisers that went through COVID-19 onboarding who can bring in guests under their supervision, and all hackerspace meetings are back in the space with remote participation supported.**
```

I'm not "fixed" on the number, just that there *should* be a number

**Review by andrewbolster — CHANGES_REQUESTED** [review]
Overall a welcome step change, but I think we should quantify "small"

File: coronavirus.md (hunk 1 above)
Comment (awflwafl) [reply coronavirus.md:7]: Ok, we should go for 15 people in that case - that is the maximum number before an additional risk assessment is required.

**Review by andrewbolster — APPROVED** [review]

**Comment by andrewbolster** [comment]
@pixelpage approved of initial draft in #management-private so merging

**Pull request merged**